src/model/extracted_licensing_info.py

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) 2022 spdx contributors
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from datetime import datetime
+from typing import Optional, List
+
+
+class CrossReference:
+    url: str
+    is_live: Optional[bool]
+    is_valid: Optional[bool]
+    is_way_back_link: Optional[bool]
+    match: Optional[str]
+    order: Optional[int]
+    timestamp: Optional[datetime]
```

Comment:
I found this in [the json schema](https://github.com/spdx/spdx-spec/blob/v2.3/schemas/spdx-schema.json#L120-L158) and [the rdf spec](https://spdx.org/rdf/ontology/spdx-2-3/#d4e2233). Since Gary stated that he uses the rdf schema as source of truth and tries to keep the tag/value spec in sync, I added them here.

@goneall can you comment whether these additional cross ref fields (everything apart from `url`) are relevant and should be handled by the Python tools?

Reply:
I do not think the Python tools need to implement these fields.  These fields have not yet been accepted as part of the official spec.  [Issue 46](https://github.com/spdx/spdx-spec/issues/46) tracks an enhancement to add these issues - which has not yet been resolved.